iOS_SDK/OneSignalSDK/Source/OneSignal.m

```diff
@@ -1536,6 +1535,17 @@ + (void)setIsOnSessionSuccessfulForCurrentState:(BOOL)value {
     isOnSessionSuccessfulForCurrentState = value;
 }
 
+static BOOL _registerUserFinished = false;
++ (BOOL)isRegisterUserFinished {
+    if (_registerUserFinished || isOnSessionSuccessfulForCurrentState) {
+        NSLog(@"ECM register finished");
+    } else {
+        NSLog(@"ECM register NOT finished");
```

Comment:
Should use `onesignalLog` instead of `NSLog`

Reply:
my fixup didn't get force pushed I removed the logs